deerlab/snlls.py

```diff
@@ -247,9 +247,9 @@ def snlls(y, Amodel, par0, lb=None, ub=None, lbl=None, ubl=None, nnlsSolver='cvx
     nonNegativeOnly = (np.all(lbl == 0)) and (np.all(np.isinf(ubl)))
 
 
+    # Use an arbitrary axis
+    ax = np.arange(1, Nlin+1)
     if includePenalty:

```

Comment:
`includePenalty` is now ambiguous - what about calling it `includeRegularization`

Reply:
I agree that `includePenalty ` is ambiguoug, however `includeRegularization` is a bit too wordy for a keyword argument.

Maybe something like `regpenalty`?

Reply:
Fine

Reply:
My bad, I thought `includePenalty` was a keyword argument, but it is just a variable. I renamed it to `includeRegularization` as you proposed.